tools/torchci/td/utils.py

```diff
@@ -161,15 +161,15 @@ def get_filtered_failed_tests() -> List[Dict[str, Any]]:
         t.name,
         t.classname,
         t.file,
-        j.head_sha,
+        j.head_sha
     FROM
-        commons.failed_tests_run t
-        join workflow_job j on t.job_id = j.id
+        default .failed_test_runs t
```

Comment:
It looks like a typo in the table name here (has a redundant space)

Reply:
This is just how they format things, the query still works

Reply:
TIL, it's a weird SQL syntax